docs/pt_BR/dependency_management.md

```diff
@@ -143,6 +138,44 @@ Get.putAsync<S>(
   bool permanent = false
 ```
 
+- Get.create:
+
+```dart
+Get.create<S>(
+  // Obrigatório: um método assíncrono que vai ser executado para instanciar sua classe
```

Comment:
Pra mim tá tudo ok, exceto esse assincrono daqui e trocar essa descrição para: "Uma função que retorna uma classe que será "fabricada" toda vez que Get.find() for chamado". Fazendo isso pode ser merged